README.md

```diff
@@ -24,17 +24,17 @@ For our in-house production, we have used manual sanding and painting for the be
 
 #### Rendered images
 
-![](images/earwig-01.png)
-![](images/earwig-02.png)
-![](images/earwig-03.png)
+<span class="images">![](https://s3.us-west-2.amazonaws.com/reference-docs-images/work_environ_mon_docs/earwig-01.png)</span>
```

Comment:
I'm getting a 403 for that s3 link:
![wem_hardware_image_issue_403](https://user-images.githubusercontent.com/197814/40447342-db1a6984-5e97-11e8-823d-483818cb0ee6.png)